controllers/integrationpipeline/integrationpipeline_adapter.go

```diff
@@ -114,6 +117,23 @@ func (a *Adapter) EnsureStatusReportedInSnapshot() (controller.OperationResult,
 		a.logger.Error(err, "Failed to update pipeline status in snapshot")
 		return controller.RequeueWithError(fmt.Errorf("failed to update test status in snapshot: %w", err))
 	}
+
+	a.logger.Info(".........status of plr", "PLR.name", a.pipelineRun.Name, "PLR.status", pipelinerunStatus)
```

Comment:
If this is debugging log entry it should be removed, if not, it should be reworded, but keep on mind it may spam logs.

Reply:
sorry, this is a result of me not pushing my latest changes to the PR.